docs/pages/database-access/auto-user-provisioning/mongodb.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: MongoDB Automatic User Provisioning 
+description: Configure automatic user provisioning for MongoDB.
+---
+
+(!docs/pages/includes/database-access/auto-user-provisioning/intro.mdx!)
+
+- Teleport cluster v14.3 or above with a configured [self-hosted
+  MongoDB](../guides/mongodb-self-hosted.mdx).
```

Comment:
```suggestion
- Teleport cluster v14.3 or above.
- A self-hosted
  MongoDB database enrolled with your Teleport cluster. Follow the [Teleport documentation](../guides/mongodb-self-hosted.mdx) to learn how to enroll your database.
```
Suggested to be more explicit about what "with a configured..." means and what the link points to.